src/MUIRichTextEditor.tsx

```diff
@@ -280,6 +288,13 @@ const MUIRichTextEditor: RefForwardingComponent<TMUIRichTextEditorRef, IMUIRichT
         start: 0,
         end: 0
     })
+    const autocompleteOptions = React.useMemo(()=>{
```

Comment:
can we rename this to `memoedAutocompleteItems`

Reply:
Resolved